tests/session/session-debug-002.phpt

```diff
@@ -55,5 +55,11 @@ object(MongoDB\Driver\Session)#%d (%d) {
   }
   ["server"]=>
   NULL
+  ["inTransaction"]=>
+  bool(true)
+  ["transactionState"]=>
+  string(9) "committed"
```

Comment:
Relevant test failure [task log](https://evergreen.mongodb.com/task_log_raw/mongo_php_driver_mongo_php7__versions~4.0_php_versions~7.3_os_php7~rhel70_test_test_replicaset_auth_patch_f317976f7487641ce355fdb101805b388920bcb8_60db6d2b7742ae77ab6f4a4b_21_06_29_18_57_48/0?type=T#L1097):

```
[2021/06/29 19:42:23.086] ========DIFF========
 [2021/06/29 19:42:23.086] 023+       ["hash"]=>
 [2021/06/29 19:42:23.086] 024+       object(MongoDB\BSON\Binary)#6 (2) {
 [2021/06/29 19:42:23.086] 025+         ["data"]=>
 [2021/06/29 19:42:23.086] 026+         string(20) "�<2�^��s��� ?!���@е"
 [2021/06/29 19:42:23.086] 027+         ["type"]=>
 [2021/06/29 19:42:23.086] 028+         int(0)
 [2021/06/29 19:42:23.086] 029+       }
 [2021/06/29 19:42:23.086] 030+       ["keyId"]=>
 [2021/06/29 19:42:23.086] 031+       int(6979303250201673730)
 [2021/06/29 19:42:23.086] 032+     }
 [2021/06/29 19:42:23.086] 036-   bool(true)
 [2021/06/29 19:42:23.086] 038-   string(9) "committed"
 [2021/06/29 19:42:23.086] 046+   bool(false)
 [2021/06/29 19:42:23.086] 048+   string(4) "none"
 [2021/06/29 19:42:23.086] ========DONE========
 [2021/06/29 19:42:23.086] FAIL MongoDB\Driver\Session debug output (after an operation) [tests/session/session-debug-002.phpt]
```

I think the problem is that you're expecting to be in a transaction and a "committed" state. The test itself never starts a transaction, so I think you should be expecting the same output as `session-debug-001.phpt`.

IIRC, the earlier part of the diff is just `run-tests.php` (the script used to execute PHPT files) outputting all of the content that matches a wildcard `%a` expectation for debugging purposes. That doesn't mean that output is incorrect, though. We typically see this when tests use _multiple_ wildcard patterns and something fails (diff output is very noisy), so I wouldn't worry about it here. Fixing the expectation for you two fields should resolve everything.